### PR TITLE
Detecte feedback loop between sampling textures and drawing framebuffer attachments

### PR DESCRIPTION
**Pull request by Richard-Yunchao**
PTAL. @zhenyao  and @kenrussell. Thanks a lot!

### CODE REVIEW

File: sdk/tests/conformance2/rendering/00_test_list.txt [hunk 1]
```diff
@@ -18,5 +18,6 @@ framebuffer-unsupported.html
 instanced-arrays.html
 --min-version 2.0.1 instanced-rendering-bug.html
 out-of-bounds-index-buffers-after-copying.html
+rendering-sampling-feedback-loop.html
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/00_test_list.txt]: We're trying to wrap up version 2.0.0 of the conformance suite. At this point I think we should put pretty much all new tests except for those for the texture sub-source entry points into 2.0.1. What do you think @Richard-Yunchao @zhenyao @jdashg ?

Comment (zhenyao) [inline sdk/tests/conformance2/rendering/00_test_list.txt]: I agree.  Let's put this to 2.0.1. Chrome will probably be able to enforce this in the initial release, so any contents violating this rule will not run.

**Comment by Richard-Yunchao** [comment]
@kenrussell , @zhenyao , agreed. I have moved this into 2.0.1. 

Could you take another look?

**Comment by zhenyao** [comment]
LGTM

**Pull request merged**